sos/report/plugins/kdump.py

```diff
@@ -114,6 +114,23 @@ class DebianKDump(KDump, DebianPlugin, UbuntuPlugin):
     files = ('/etc/default/kdump-tools',)
     packages = ('kdump-tools',)
 
+    option_list = [
+        PluginOpt("get-vm-core", default=False, val_type=bool,
+                  desc="collect vm core")
+    ]
+
+    def read_kdump_conffile(self):
+        """ Parse /etc/default/kdump-tools """
+        path = "/var/crash"
+
+        kdump = '/etc/default/kdump-tools'
+        with open(kdump, 'r', encoding='UTF-8') as file:
+            for line in file:
+                if line.startswith("KDUMP_COREDIR"):
```

Comment:
May want to check for spaces at the beginning of the line

Reply:
Good point about trimming spaces first. Thanks for the review.